plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/style.scss

```diff
@@ -44,3 +38,15 @@
 		}
 	}
 }
+
+@include breakpoint(">=782px") {
+	.wc-block-cart .wc-block-cart__submit-container--sticky {
+		display: none;
+	}
+}
+
+@media screen and (max-height: 500px) {
```

Comment:
I think we should have min-height here. As it is now removing the sticky container for for mobile screens as well.

Reply:
It's expected that the sticky button will disappear on mobile screens when the screen height is smaller than 500px, to ensure that the sticky button does not overlap the product line items. How would you use `min-height` in this case? This part is not clear to me.

Reply:
Please ignore my comment. I thought it was related to the screen orientation and maximum height, so I tested it on a mobile device as well.